server/pkg/proxystore/proxystore_test.go

```diff
@@ -77,3 +78,45 @@ func Example() {
 	// - IPs:{V4:"10.0.0.2"} [  ]
 
 }
+
+// TestSessionAffinitySetClientIP creates scenario to validate SessionAffinity
+// Ref: https://github.com/kubernetes-sigs/kpng/issues/156
+func TestSessionAffinitySetClientIP(t *testing.T) {
```

Comment:
`func UnCommaneMeAmimTestSessionAffinitySetClientIP(t *testing.T) {`

Reply:
then we'll merge #171 right after :), and Uncomment this test.

Reply:
sure thing